src/externalized/mercs/MercProfile.cc

```diff
@@ -139,7 +140,7 @@ std::unique_ptr<MERCPROFILESTRUCT> MercProfile::deserializeStruct(const JsonValu
 	}
 	prof->ubCivilianGroup = Internals::getCivilianGroupEnumFromString(r.getOptionalString("civilianGroup"));
 	if (r.getOptionalBool("isGoodGuy")) {
-		prof->ubMiscFlags |= PROFILE_MISC_FLAG3_GOODGUY;
+		prof->ubMiscFlags3 |= PROFILE_MISC_FLAG3_GOODGUY;
```

Comment:
What's this about? Looks unrelated to the rest of the changes.

Reply:
Quick typo fix. Buns currently doesn't have weapons displayed on the AIM site because of this.